common/datavalues/src/types/serializations/boolean.rs

```diff
@@ -17,33 +17,12 @@ use common_exception::Result;
 use common_io::prelude::*;
 
 use crate::prelude::*;
-use crate::TypeSerializer;
 
-pub struct BooleanSerializer {
+pub struct BooleanDeserializer {
     pub builder: BooleanArrayBuilder,
 }
 
-impl TypeSerializer for BooleanSerializer {
-    fn serialize_strings(&self, column: &DataColumn) -> Result<Vec<String>> {
-        let array = column.to_array()?;
-        let array: &DFBooleanArray = array.static_cast();
-
-        let result: Vec<String> = array
-            .into_iter()
-            .map(|x| {
-                x.map(|v| {
-                    if v {
-                        "true".to_owned()
-                    } else {
-                        "false".to_owned()
-                    }
-                })
-                .unwrap_or_else(|| "NULL".to_owned())
-            })
-            .collect();
-        Ok(result)
-    }
-
+impl TypeDeserializer for BooleanDeserializer {
```

Comment:
Seems `TypeDeserializer` and `TypeSerializer` are both ok, because we have 

`serialize` and `de` (`deserialize`) method, what about making it a new name, like :  `SerDe`